raster/src/main/scala/geotrellis/raster/render/ColorMap.scala

```diff
@@ -228,8 +230,15 @@ class IntColorMap(breaksToColors: Map[Int, Int], val options: Options = Options.
 
   def cache(h: Histogram[Int]): ColorMap = {
     val ch = h.mutable
-    h.foreachValue(z => ch.setItem(z, map(z)))
-    new IntCachedColorMap(colors, ch, options)
+    val cachedColors = h.values()
+    cfor(0)( _ < cachedColors.length, _ + 1) { i =>
```

Comment:
`h.totalCount` would give me the total number of times any value was seen bytes histogram across all buckets. Not sure if thats useful here. What are you thinking?